apps/obsidian/.gitignore

```diff
@@ -0,0 +1,2 @@
+plugin
+main.js
```

Comment:
I'm guessing this was from the first `.env` folder setting?  Let's remove it if it is no longer necessary.

Reply:
oh these files come from turbo dev. maybe we can just commit them once?

Reply:
That is interesting.  I am not replicating this on my end.  Could you delete them and try running `turbo dev` again, this time with the updated Obsidian `.env` file?  If they are still getting created we can look into the issue further at tomorrow's standup.

Regardless, let's remove them from this PR.